src/sentry_core.c

```diff
@@ -18,6 +18,7 @@
 #include "sentry_tracing.h"
 #include "sentry_transport.h"
 #include "sentry_value.h"
+#include "transports/sentry_disk_transport.h"
```

Comment:
This is no longer necessary, right? I would love to keep direct dependencies from core to concrete implementations at a minimum.